File: RetroGameGauntlet.Droid/Adapters/WebLauncherAdapter.cs
using Android.App;
using Android.Support.CustomTabs;
using Android.Support.V4.Content;
using Plugin.CurrentActivity;
using RetroGameGauntlet.PCL.Adapters;
using Uri = Android.Net.Uri;

namespace RetroGameGauntlet.Droid.Adapters
{
    public class WebLauncherAdapter : IWebLauncherAdapter
    {
        private Activity Activty => CrossCurrentActivity.Current.Activity;

        public WebLauncherAdapter()
        {
        }

        public void OpenUri(string url)
        {
            CustomTabsIntent.Builder builder = new CustomTabsIntent.Builder();
            builder.SetToolbarColor(ContextCompat.GetColor(Activty, Resource.Color.primary));
            CustomTabsIntent customTabsIntent = builder.Build();
            customTabsIntent.LaunchUrl(Activty, Uri.Parse(url));
        }
    }
}

File: RetroGameGauntlet.Droid/Adapters/WebLauncherAdapter.fs
namespace RetroGameGauntlet.Droid.Adapters

open Android.Support.CustomTabs
open Android.Support.V4.Content
open Plugin.CurrentActivity
open RetroGameGauntlet.Droid
open RetroGameGauntlet.Core.Adapters

type WebLauncherAdapter () =
    let activity = CrossCurrentActivity.Current.Activity

    interface IWebLauncherAdapter with
        member this.OpenUri url = 
            let builder = new CustomTabsIntent.Builder()
            builder.SetToolbarColor(ContextCompat.GetColor(activity, Resources.Color.primary)) |> ignore
            let customTabsIntent = builder.Build()
            customTabsIntent.LaunchUrl(activity, Android.Net.Uri.Parse(url))